heareval/predictions/task_predictions.py

```diff
@@ -997,49 +1121,77 @@ def print_scores(grid_point_results: List[GridPointResult]):
         grid_point_results.append(grid_point_result)
         print_scores(grid_point_results)
 
-    # Use the best model to compute test scores
+    # Use the best model to train remaining splits and compute test scores
```

Comment:
This isn't totally accurate -- should be "Use the best hyperparameters to train models for remaining folds, then compute test scores using the resulting models"